Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -4072,6 +4091,18 @@ class CloneDeserializer : CloneBase {
             uint32_t width;
             if (!read(width))
                 return JSValue();
+            if (width == ImageDataPoolTag) {
+                unsigned index = 0;
```

Comment:
Seems like this would be better using `std::optional` instead of an out argument.